ssz.go

```diff
@@ -0,0 +1,169 @@
+package ssz
```

Comment:
No need to review this file, simply collected `Marshal, Unmarshal, HashTreeRoot, SigningRoot` into a single file